expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -102,8 +102,12 @@ public ExpandedOrganization getOrganization(Entity dataEntry) throws NotFoundExc
                                       : resourceService.getResourceByIdentifier(ticketEntry.getResourceIdentifier())
                                           .getResourceOwner().getOwnerAffiliation();
 
+            var organizationIdentifier = Objects.nonNull(organizationId)
+                                             ? UriWrapper.fromUri(organizationId).getLastPathElement()
+                                             : "";
```

Comment:
What about null instead of empty string?

Can be wrapped in optional of nullable:
```
var organizationIdentifier = Optional.ofNullable(organizationId)
                                             .map(UriWrapper::fromUri)
                                             .map(UriWrapper::getLastPathElement)
                                             .orElse(null);
```

Reply:
Good catch! Will do